admin/app/assets/config/solidus_admin_manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
+//= link_tree ../stylesheets .css
+//= link_tree ../../javascript .js
```

Comment:
Shouldn't it be `../` instead of `../../`? Also, Rails uses `javascripts` (pluralized) as the default for new applications. Should we use the same convention?

Reply:
This is the default for new apps when you use importmaps (which is also default), I think the rationale is that app/assets is the realm of sprockets/propshaft and app/javascript is where you have the source files that will be managed by your JS bundler (in this case `importmap-rails`).

```
⤑ rails new foobar                                                          /tmp
…[CUT]
⤑ cd foobar                                                                                                                                                                                                /tmp
foobar:master ⤑ tree app                                                                                                                                                                              /t/foobar
app
├── assets
│   ├── config
│   │   └── manifest.js
│   ├── images
│   └── stylesheets
│       └── application.css
├── channels
│   └── application_cable
│       ├── channel.rb
│       └── connection.rb
├── controllers
│   ├── application_controller.rb
│   └── concerns
├── helpers
│   └── application_helper.rb
├── javascript
│   ├── application.js
│   └── controllers
│       ├── application.js
│       ├── hello_controller.js
│       └── index.js
├── jobs
│   └── application_job.rb
├── mailers
│   └── application_mailer.rb
├── models
│   ├── application_record.rb
│   └── concerns
└── views
    └── layouts
        ├── application.html.erb
        ├── mailer.html.erb
        └── mailer.text.erb

17 directories, 16 files
```

Reply:
Oh, cool :+1: